test-forms/index.json

```diff
@@ -528,7 +528,7 @@
   { "html": "leafly_login.html" },
   { "html": "etsy_security-step.html" },
   { "html": "github_assign-reviewer.html" },
-  { "html": "simplii_login.html" },
+  { "html": "simplii_login.html", "expectedFailures": ["unknown"] },
```

Comment:
There's an empty field in a password group, which is most likely some type of password field.